sirepo/package_data/static/js/sirepo-plotting.js

```diff
@@ -244,7 +246,7 @@ class PlotPolygon extends AbstractPlotShape2D {
     getSizeCoords() {
         return this.getCoords(this.size);
     }
-
+    
     setPoints(arr) {
```

Comment:
@mkeilman can you check if your editor supports "remove whitespaces in blank lines" feature? Maybe something along these lines:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/205803079-Configure-editor-to-remove-whitespaces-in-blank-lines

Reply:
Yes - I've turned that on. Also turned on "ensure every saved file ends with a line break" while I was at it